package.json

```diff
@@ -8,7 +8,12 @@
     "test": "jest"
   },
   "dependencies": {
+    "@types/aws-lambda": "^8.10.64",
```

Comment:
do you need all these packages in the root and import-service package.json?

Reply:
Will clean it up later, it need for some dirty tests =)